themes/default/content/templates/serverless-application/aws/index.md

```diff
@@ -0,0 +1,133 @@
+$ pulumi up
+```
+
+When the deployment completes, Pulumi exports the following [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) values:
+
+url
+: The HTTP URL for the application.
+
+Stack outputs are useful in a number of ways, most commonly as inputs to other stacks or cloud resources.
+
+## Customizing the project
+
+Projects created with the AWS Serverless Application template expose the following [configuration]({{< relref "/docs/intro/concepts/config" >}}) settings:
+
+path
+: The path to the folder containing the files of the website. Defaults to `www`, which is the name (and relative path) of the folder included with the template. The /date path is a GET method that retrieves the current time from the Lambda function.
+
+code
+: References the folder containing the Lambda function code, which is in a file called `handler`
+
+None of these settings is required; by default, the AWS Serverless Application template is set up to deploy the website using the files contained in the `www` and `function` folders that are bundled with the template.
+
+### Using your own web content
+
+If you already have a website you'd like to deploy, you can do so by replacing contents of the `www` folder and redeploying with `pulumi up`.
+
+Alternatively, you can configure the stack to deploy from another folder on your machine by using [`pulumi config set`]({{< relref "/docs/reference/cli/pulumi_config_set" >}}) to change the value of the `path` setting:
+
+```bash
+$ pulumi config set path ../my-website/dist
+$ pulumi up
+```
+
+### Customizing the application's functionality
+
+You can customize the AWS serverless application's functionality by editing the Lambda function to perform another action, such as displaying a countdown until a specific time in the future. You could also add new functionality by creating a new Lambda function, adding a new path to the REST API, and updating the HTML script to call the new path.
+
+## Next steps
+
+Templated projects are meant to be customized, and every web project comes with its own unique set of needs. This section includes a few examples aimed at helping you to adapt your new project to address some of the more common ones.
+
+### Adding a database
+
+(...)
+
+### Cleaning up
+
+You can cleanly destroy the stack and all of its infrastructure with [`pulumi destroy`]({{< relref "/docs/reference/cli/pulumi_destroy" >}}):
+
+```bash
+$ pulumi destroy
+```
+
+## Learn more
+
+Congratulations! You're now well on your way to managing a production-grade AWS serverless application with Pulumi --- and there's lots more you can do from here:
+
+* Discover more architecture templates as they're available in [Templates &rarr;]({{< relref "/templates" >}})
+* Dive into the AWS package by exploring the [API docs in the Registry &rarr;]({{< relref "/registry/packages/aws" >}})
+* Expand your understanding of how Pulumi works in [Architecture &amp; Concepts &rarr;]({{< relref "/docs/intro/concepts" >}})
```

Comment:
I really don't think this is a good CTA. You should direct them over to Pulumi Learn pathways